src/util/Bundles.js

```diff
@@ -141,10 +138,7 @@ define([
     // zh-CN -> zh_CN
     languageCode = language.replace('-', '_');
 
-    path = assets.rewrite(bundlePathTpl({
-      bundle: bundle,
-      languageCode: languageCode
-    }));
+    path = assets.rewrite(`/labels/json/${bundle}_${languageCode}.json`);
```

Comment:
shall it run through `encodeURI` ?

Reply:
It probably won't hurt. I'll give it a try.